.ocamlformat

```diff
@@ -9,8 +10,7 @@ break-sequences=true
 cases-exp-indent=2
 doc-comments=before-except-val
 dock-collection-brackets=false
-if-then-else=k-r
-parens-tuple-patterns=always
+if-then-else=keyword-first
```

Comment:
Oooh, I disagree! Yuck!

Reply:
Pick your poison from currently supported options ^^! I don't mind either way.

Reply:
Yup, I don't like it either but I don't think we have a choice. RIP K&R style.

Reply:
Maybe the default `compact` style is preferable?